packages/core/src/App/Containers/Layout/header/header.tsx

```diff
@@ -1,24 +1,26 @@
 import React from 'react';
+import { useLocation } from 'react-router-dom';
 import { PlatformContext, routes } from '@deriv/shared';
+import { observer, useStore } from '@deriv/stores';
 import DefaultHeader from './default-header.jsx';
 import DashboardHeader from './dashboard-header.jsx';
-import TradingHubHeader from './trading-hub-header.jsx';
 import DTraderHeader from './dtrader-header.jsx';
-import { connect } from 'Stores/connect';
-import { useLocation } from 'react-router-dom';
+import TradersHubHeader from './traders-hub-header';
 
-const Header = ({ is_logged_in }) => {
+const Header = observer(() => {
+    const { client } = useStore();
+    const { is_logged_in } = client;
     const { is_appstore } = React.useContext(PlatformContext);
```

Comment:
do we need this is appstore?

Reply:
@matin-deriv asked to leave it for another card.